tests/test_models.py

```diff
@@ -0,0 +1,84 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+
+from taxi.models import Manufacturer, Driver, Car
+
+
+class TestManufactureModel(TestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.manufacturer1 = Manufacturer.objects.create(
+            name="Test_name",
+            country="Test_Country"
+        )
+        cls.manufacturer2 = Manufacturer.objects.create(
+            name="Best_name",
+            country="Test_Country"
+        )
+        cls.manufacturer3 = Manufacturer.objects.create(
+            name="Rest_name",
+            country="Test_Country"
+        )
+
+    def test_manufacturer_representation(self):
+        self.assertEqual(str(self.manufacturer1), "Test_name Test_Country")
+
+    def test_manufacturer_ordering(self):
+        manufacturers = Manufacturer.objects.all()
+        self.assertEqual(list(manufacturers), [
+            self.manufacturer2,
+            self.manufacturer3,
+            self.manufacturer1])
```

Comment:
The `test_manufacturer_ordering` test assumes a specific ordering of manufacturers. Ensure that the `Manufacturer` model has an ordering defined in its `Meta` class that matches this assumption. Otherwise, this test may fail if the ordering is not explicitly set.